src/language-js/print/jsx.js

```diff
@@ -126,7 +126,7 @@ function printJsxElementInternal(path, options, print) {
 
   const rawJsxWhitespace = options.singleQuote ? "{' '}" : '{" "}';
   const jsxWhitespace = isMdxBlock
-    ? " "
+    ? line
```

Comment:
I'm worrying that it might not be just a careless mistake. I'm not familiar with MDX at all...